src/index.js

```diff
@@ -2,81 +2,37 @@ const axios = require('axios');
 const helpers = require('./helpers');
 
 const headers = {
-	'Accept': 'application/vnd.codebottle.v1+json'
+  Accept: 'application/vnd.codebottle.v1+json',
 };
 
 module.exports = {
-	search: (keywords, language) => {
```

Comment:
What program and config did you use for linting this?
I don't think applying what you think is 'sexier' is appropriate. This project already has a style, albeit not dicumented.

Reply:
It’s eslint and v0id config 

I replaced it with new style as it’s easier to maintain and read
Also it complies with config rules